packages/wonder-blocks-popover/__tests__/__snapshots__/generated-snapshot.test.js.snap

```diff
@@ -1350,6 +1351,7 @@ exports[`wonder-blocks-popover example 9 1`] = `
     <button
       aria-label="Close Popover"
       className=""
+      data-test-id="popover-close-btn"
```

Comment:
Just one minor request here. Could you please add a `testId` prop to this example to verify that we are overriding the `data-test-id` value properly?

You can modify that example in: https://github.com/Khan/wonder-blocks/blob/master/packages/wonder-blocks-popover/components/popover-content-core.md#L41